perception/map_based_prediction/config/map_based_prediction.param.yaml

```diff
@@ -27,6 +27,9 @@
       timeout_set_for_no_intention_to_walk: [1.0, 0.0] # [s] values of map
     # parameter for shoulder lane prediction
     prediction_time_horizon_rate_for_validate_shoulder_lane_length: 0.8
+    use_crosswalk_user_history:
```

Comment:
I do not want these parameter to be `true` in default. 
- These association and object keeping process are handled within some tracking processes. I think post processing in prediction should be a option to prevent confusion.
- FYI: After update in tracker process, [PR6343](https://github.com/autowarefoundation/autoware.universe/pull/6343) for example, pedestrian tracking will much stable than older version. Centerpoint and yolo model update will also improve stability.

Reply:
I agree to switch it off by default, because current stabilization process is not very elaborate. But in general I think map based prediction should not rely on perfect tracking. There should be multiple layers of protection in these safety critical flows such as pedestrian avoidance